ios/FluentUI/Drawer/DrawerTokenSet.swift

```diff
@@ -8,86 +8,61 @@ import UIKit
 /// Design token set for the `Drawer` control
 public class DrawerTokenSet: ControlTokenSet<DrawerTokenSet.Tokens> {
     public enum Tokens: TokenSetKey {
-        /// `ShadowInfo` for the shadow used in the `Drawer` control.
-        case shadow
-
-        /// Color used when dimming the background while showing the `Drawer` control.
-        case backgroundDimmedColor
+        /// Corner radius for the popover style `Drawer` control.
+        case cornerRadius
 
         /// Color used for the background of the content of the `Drawer` control.
         case drawerContentBackground
 
-        /// Color used for the background of the popover style `Drawer` control.
-        case popoverContentBackground
-
         /// Color used for the navigation bar of the `Drawer` control.
         case navigationBarBackground
 
-        /// Corner radius for the popover style `Drawer` control.
-        case cornerRadius
-
-        /// Minimum horizontal margin for the `Drawer` control.
-        case minHorizontalMargin
-
-        /// Minimum vertical margin for the `Drawer` control.
-        case minVerticalMargin
+        /// Color used for the background of the popover style `Drawer` control.
+        case popoverContentBackground
 
-        /// Offset of the shadow for the `Drawer` control.
-        case shadowOffset
+        /// `ShadowInfo` for the shadow used in the `Drawer` control.
+        case shadow
     }
 
     init() {
         super.init { token, theme in
             switch token {
-            case .shadow:
-                return .shadowInfo({
-                    ShadowInfo(keyColor: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0.03),
-                                                      dark: ColorValue(r: 0, g: 0, b: 0, a: 0.07)),
-                               keyBlur: 14,
-                               xKey: 0,
-                               yKey: 14,
-                               ambientColor: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0.33),
-                                                          dark: ColorValue(r: 0, g: 0, b: 0, a: 0.66)),
-                               ambientBlur: 4,
-                               xAmbient: 0,
-                               yAmbient: 0)
-                })
-
-            case .backgroundDimmedColor:
-                return .dynamicColor({
-                    DynamicColor(light: ColorValue(r: 0.0, g: 0.0, b: 0.0, a: 0.66),
-                                 dark: ColorValue(r: 0.0, g: 0.0, b: 0.0, a: 0.99))
-                })
+            case .cornerRadius:
+                return .float({ 14 })
```

Comment:
Does the design not use a corner radius from global tokens?

Reply:
Yep just fixed it